src/Book.java

```diff
@@ -0,0 +1,27 @@
+public class Book {
+    String name;
+    Author author;
+    int yearPublication;
```

Comment:
```suggestion
    private String name;
    private Author author;
    private int yearPublication;
```
поля всегда должны быть приватные, если есть геттеры/сеттеры, про это в задании написано